src/Sylius/Component/Addressing/Model/Zone.php

```diff
@@ -170,7 +171,6 @@ public function removeMember(ZoneMemberInterface $member)
     {
         if ($this->hasMember($member)) {
             $this->members->removeElement($member);
-            $member->setBelongsTo(null);
```

Comment:
Hmm, in fact this could be wrong, but can zone member don't belong to any zone? How would be this listed?